src/main/java/Duke.java

```diff
@@ -1,10 +1,48 @@
+import java.util.Scanner;
+//import java.util.ArrayList;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+        int endMain=0;
+        //ArrayList<String> split=new ArrayList<>();
```

Comment:
You could remove commented code to improve overall code readability. There are several of such lines.